packages/esm-patient-search-app/src/compact-patient-search/compact-patient-banner.scss

```diff
@@ -23,13 +25,16 @@
 
 .patientName {
   @include type.type-style('heading-02');
+  color: $text-02;
 }
 
 .patientAvatar {
   width: spacing.$spacing-09;
   height: spacing.$spacing-09;
   margin: spacing.$spacing-03 spacing.$spacing-05 spacing.$spacing-03 0;
   border-radius: 1px;
+  font-weight: unset;
+  line-height: unset;
```

Comment:
Unsets some styling from https://github.com/openmrs/openmrs-esm-core/blob/8bf18b44dc900de42b485e85c408bad354ac4f25/packages/apps/esm-primary-navigation-app/src/components/navbar/navbar.scss#L9 that affects the appearance of the text in the patient avatar.

Reply:
Great!